service/resource/cloudformation/load_balancer_test.go

```diff
@@ -1,4 +1,4 @@
-package legacy
+package cloudformation
```

Comment:
I don't think we should move logic from the legacy package to the cloud formation resource. I think it is better to treat legacy as frozen and add the new code to the operatorkit resources.

IMO some minor changes for enabling / disabling logic are OK. In this case we could move LoadBalancerName to the key package.

Reply:
Yes, but IMO when possible it would be better to have only one code path, if we duplicate code during the transition it can be hard to track problems. In this case I've moved the logic that generates the load balancer name to the key package as you suggest later and that code is used for both legacy and cloudformation, let me know WDYT.